selfdrive/boardd/usbdevice.h

```diff
@@ -0,0 +1,44 @@
+#pragma once
+
+#include <atomic>
+#include <mutex>
+#include <vector>
+
+#include <libusb-1.0/libusb.h>
+
+#define TIMEOUT 0
+#define PANDA_VENDOR_ID 0xBBAA
+#define PANDA_PRODUCT_ID 0xDDCC
```

Comment:
these should go in a header in panda

Reply:
Will need to refactor panda headers slightly tomorrow, also will move other defines like RECV_SIZE etc.
Think bus count also should be included from panda?

Reply:
yeah that sounds good